views/wishCardFullPage.ejs

```diff
@@ -40,44 +44,39 @@
-                        <a class="dropdown-item" href="#">Happy Birthday to the sweetest kid in the entire world.</a>
-                        <a class="dropdown-item" href="#">Happy birthday to a future superstar!</a>
-                        <a class="dropdown-item" href="#">Happy birthday, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#">Merry Christmas, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#">Happy holidays, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, you are awesome!</a>
-                        <a class="dropdown-item" href="#">Lots of love and best wishes, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, hope you enjoy my gift!</a>
-                        <a class="dropdown-item" href="#">Merry Christmas and a Happy New Year</a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, have a happy holiday</a>
-                        <a class="dropdown-item" href="#">Congratulations, <%= wishcard.childFirstName %></a>
-                    </div>
-                </div>
-
-                <input type="submit" id="submit" class="mt-4" value="Send Message">
-
+                
+                <form class="message" action="/wishcards/message" method="post">
+                    <select class="custom-select">
+                        <option selected>Messages</option>
+                        <option value="1"><%= user.fName %> sends you love, <%= wishcard.childFirstName %></option>
+                        <option value="2">Happy Birthday to the sweetest kid in the entire world.</option>
+                        <option value="3">Happy birthday to a future superstar!</option>
+                        <option value="4">Happy birthday, <%= wishcard.childFirstName %></option>
+                        <option value="5">Merry Christmas, <%= wishcard.childFirstName %></option>
+                        <option value="6">Happy holidays, <%= wishcard.childFirstName %></option>
+                        <option value="7"><%= wishcard.childFirstName %>, you are awesome!</option>
+                        <option value="8">Lots of love and best wishes, <%= wishcard.childFirstName %></option>
+                        <option value="9"><%= wishcard.childFirstName %>, hope you enjoy my gift!</option>
+                        <option value="10">Merry Christmas and a Happy New Year</option>
+                        <option value="11"><%= wishcard.childFirstName %>, have a happy holiday</option>
+                        <option value="12">Congratulations, <%= wishcard.childFirstName %></option>
+                    </select>
+                    <input type="submit" id="submit" class="mt-4" value="Send Message">
+                </form>
+                
```

Comment:
I would suggest to generate an array in the backend and just iterate over it here, gives us more flexibilty and cleans up the code. maybe we can add a placeholder in the strings like %name% and replace every occurence of the placeholder with the name.

Not neccessary though